examples/user_guide/07-Tabular_Datasets.ipynb

```diff
@@ -487,7 +487,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Data is often grouped in various ways, and the Dataset interface provides various means to easily compare between groups and apply statistical aggregates. We'll start by generating some synthetic data with two groups along the x-axis and 4 groups along the y axis."
+    "Data is often grouped in various ways, and the Dataset interface provides various means to easily compare between groups and apply statistical aggregates. We'll start by generating some synthetic data with two groups along the x-axis and 4 groups along the y-axis."
```

Comment:
Right.  I try to train people about [how to use hyphens](https://homepages.inf.ed.ac.uk/jbednar/writingtips.html#hyphens), but no one cares about it but me. :-)